app/models/archival_media_bag_parser.rb

```diff
@@ -36,8 +36,45 @@ def pbcore_parser_for_barcode(barcode)
     pbcore_parsers.find { |pbcore| pbcore.barcode == barcode }
   end
 
+  def image_file(barcode:)
```

Comment:
Can you please add a unit test to the bag parser spec since this is a public method